client/src/mortgageComponents/ads/AdStyles/Bold.jsx

```diff
@@ -0,0 +1,9 @@
+import Styled from 'styled-components';
+
+const Bold = Styled.span`
```

Comment:
This definitely works, however I wonder if you can just use the <strong> tag to render bold characters or declaring a font-weight in a "Seller" styling module, instead of making a whole module for bolding. But then again, the text in these cards is minimal so any way works!

Reply:
I ended up setting bold separately since the site renders rates in bold but not the %, that is why I separated out the Bold tag. However I probably should move it to a global style at some point.